great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -16,27 +16,44 @@
 from great_expectations.core.batch_spec import (
     AzureBatchSpec,
     BatchSpec,
+    GCSBatchSpec,
     PathBatchSpec,
     RuntimeDataBatchSpec,
     S3BatchSpec,
 )
-from great_expectations.core.util import AzureUrl, S3Url, sniff_s3_compression
+from great_expectations.core.util import AzureUrl, GCSUrl, S3Url, sniff_s3_compression
 from great_expectations.execution_engine import ExecutionEngine
 from great_expectations.execution_engine.execution_engine import MetricDomainTypes
 from great_expectations.execution_engine.pandas_batch_data import PandasBatchData
 
+logger = logging.getLogger(__name__)
```

Comment:
Moved logger up before importing cloud providers in order to use it for `logger.debug`